ValheimPlus/Configurations/Sections/PlayerConfiguration.cs

```diff
@@ -8,6 +8,7 @@ public class PlayerConfiguration : ServerSyncConfig<PlayerConfiguration>
         public bool disableCameraShake { get; internal set; } = false;
         public float baseUnarmedDamage { get; internal set; } = 0;
         public bool cropNotifier { get; internal set; } = false;
-
+        public float restSecondsPerComfortLevel { get; internal set; } = 0;
```

Comment:
I think this value would be better represented as its default float `= 60` and people can update it to what they like instead of a percentage modifier